[2025-08-18T20:27:00.896+0000] {taskinstance.py:1979} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: customer_churn_pipeline.data_validation scheduled__2025-08-17T00:00:00+00:00 [queued]>
[2025-08-18T20:27:00.908+0000] {taskinstance.py:1979} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: customer_churn_pipeline.data_validation scheduled__2025-08-17T00:00:00+00:00 [queued]>
[2025-08-18T20:27:00.909+0000] {taskinstance.py:2193} INFO - Starting attempt 1 of 1
[2025-08-18T20:27:00.927+0000] {taskinstance.py:2217} INFO - Executing <Task(BashOperator): data_validation> on 2025-08-17 00:00:00+00:00
[2025-08-18T20:27:00.934+0000] {standard_task_runner.py:60} INFO - Started process 282 to run task
[2025-08-18T20:27:00.939+0000] {standard_task_runner.py:87} INFO - Running: ['***', 'tasks', 'run', 'customer_churn_pipeline', 'data_validation', 'scheduled__2025-08-17T00:00:00+00:00', '--job-id', '5', '--raw', '--subdir', 'DAGS_FOLDER/customer_churn_pipeline_dag.py', '--cfg-path', '/tmp/tmp2yaw7ax1']
[2025-08-18T20:27:00.942+0000] {standard_task_runner.py:88} INFO - Job 5: Subtask data_validation
[2025-08-18T20:27:01.021+0000] {task_command.py:423} INFO - Running <TaskInstance: customer_churn_pipeline.data_validation scheduled__2025-08-17T00:00:00+00:00 [running]> on host 2e4674d7276b
[2025-08-18T20:27:01.115+0000] {taskinstance.py:2513} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='customer_churn_pipeline' AIRFLOW_CTX_TASK_ID='data_validation' AIRFLOW_CTX_EXECUTION_DATE='2025-08-17T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2025-08-17T00:00:00+00:00'
[2025-08-18T20:27:01.118+0000] {subprocess.py:63} INFO - Tmp dir root location: /tmp
[2025-08-18T20:27:01.120+0000] {subprocess.py:75} INFO - Running command: ['/usr/bin/bash', '-c', 'cd /opt/***/Customer_Churn_Prediction_Pipeline && python src/data_validation.py']
[2025-08-18T20:27:01.133+0000] {subprocess.py:86} INFO - Output:
[2025-08-18T20:27:02.512+0000] {subprocess.py:93} INFO - === DATA VALIDATION REPORT ===
[2025-08-18T20:27:02.513+0000] {subprocess.py:93} INFO - === END OF REPORT ===
[2025-08-18T20:27:02.660+0000] {subprocess.py:97} INFO - Command exited with return code 0
[2025-08-18T20:27:02.716+0000] {taskinstance.py:1149} INFO - Marking task as SUCCESS. dag_id=customer_churn_pipeline, task_id=data_validation, execution_date=20250817T000000, start_date=20250818T202700, end_date=20250818T202702
[2025-08-18T20:27:02.770+0000] {local_task_job_runner.py:234} INFO - Task exited with return code 0
[2025-08-18T20:27:02.831+0000] {taskinstance.py:3312} INFO - 1 downstream tasks scheduled from follow-on schedule check
